code/game/objects/items/weapons/tools/multitool.dm

```diff
@@ -40,7 +40,6 @@
 	degradation = 2
 	workspeed = 1.6
 	max_upgrades = 5 //UNLIMITED MODS!
-	suitable_cell = /obj/item/cell/medium
```

Comment:
why are you making it take a small cell?

Reply:
Trilby, dear, I made the greyson multitool a children of the ADVANCED multitool, it should inherit the medium cell from it

Reply:
Unless the whole block of Greyson multitool code has to be written BELOW the advanced multitool one?